README.md

```diff
@@ -36,12 +36,29 @@ mailing list.
 
 ## Installation
 
+### Bazel
+
+If you build with `bazel`, follow the [`bazel` documenation for referencing
```

Comment:
s/documenation/documentation/

Reply:
Whoops, I'll fix this (and your comment below) in a follow up